.git-hooks/hooks/pull-update-deps

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# MIT © Original by Sindre Sorhus - https://gist.github.com/sindresorhus/7996717
+changed_files="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)"
+
+check_run() {
+	printf "$changed_files" | grep -q "$1" && eval "$2"
+}
+
+# check for npm pkgs change
+check_run package.json "npm install"
```

Comment:
We primarily use `yarn` rather than `npm` to manager local dependencies. We also find that sometimes (depending on the OS) we have to use an extra flag, `yarn --ignore-engines`.

Reply:
This has been addressed.